android/java/org/chromium/chrome/browser/vpn/utils/BraveVpnPrefUtils.java

```diff
@@ -160,8 +160,14 @@ public static void setPurchaseToken(String value) {
         SharedPreferences.Editor sharedPreferencesEditor = mSharedPreferences.edit();
         sharedPreferencesEditor.putString(PREF_BRAVE_VPN_PURCHASE_TOKEN, value);
         sharedPreferencesEditor.apply();
+        String[] purchaseTokenParts = value.split(".");
+        String purchaseToken = (purchaseTokenParts.length >= 2
```

Comment:
is it ever more than 2? if not should just check `==2`. otherwise i think we should check every token part is base64 encoded, not just the first 2.

Reply:
Fixed 😄